tests/ui/cfg/true-false.rs

```diff
@@ -1,7 +1,6 @@
 //@ run-pass
 
 #![feature(link_cfg)]
-#![feature(cfg_boolean_literals)]
 
 #[cfg(true)]
```

Comment:
Question: I thought of another case where a test may be useful: what's the behavior of

```rs
#[cfg(true)]
#[cfg(false)]
fn main() {}
```

vs

```rs
#[cfg(false)]
#[cfg(true)]
fn main() {}
```

Reply:
It's the same, `main` would not be defined. As the `#[cfg]` attribute only expands when the condition evaluates to true.